applications/knative/1.19.5/extra-images.txt

```diff
@@ -14,6 +14,8 @@ gcr.io/knative-releases/knative.dev/eventing/cmd/jobsink:v1.19.5
 gcr.io/knative-releases/knative.dev/eventing/cmd/mtchannel_broker:v1.19.5
 gcr.io/knative-releases/knative.dev/eventing/cmd/mtping:v1.19.5
 gcr.io/knative-releases/knative.dev/eventing/cmd/webhook:v1.19.5
+gcr.io/knative-releases/knative.dev/net-istio/cmd/controller:1.19.6
+gcr.io/knative-releases/knative.dev/net-istio/cmd/webhook:1.19.6
```

Comment:
Two things:
- We're missing a leading `v` in the images. The image pulls here don't work without them:
```
docker pull gcr.io/knative-releases/knative.dev/net-istio/cmd/controller:1.19.6
Error response from daemon: manifest for gcr.io/knative-releases/knative.dev/net-istio/cmd/controller:1.19.6 not found: manifest unknown: Failed to fetch "1.19.6"
```
- The other components use a `v1.19.5` tag.
```suggestion
gcr.io/knative-releases/knative.dev/net-istio/cmd/controller:v1.19.5
gcr.io/knative-releases/knative.dev/net-istio/cmd/webhook:v1.19.5
```

Reply:
In a followup conversation, the image pull SHA's match with `v1.19.6` tags.